core/management/commands/queue_warm_cache.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
+
+from chronam.core.models import Page
+from chronam.core import tasks
+
+_logger = logging.getLogger(__name__)
```

Comment:
This should follow PEP-8 for a module-level variable. I like `LOGGER` for this.

Reply:
Also, there should be two lines before a function so I'd run `pycodestyle` on this or just go straight to `autopep8`

Reply:
```diff
$ autopep8 --diff core/management/commands/queue_warm_cache.py 
--- original/core/management/commands/queue_warm_cache.py
+++ fixed/core/management/commands/queue_warm_cache.py
@@ -8,14 +8,15 @@
 
 _logger = logging.getLogger(__name__)
 
+
 class Command(BaseCommand):
     help = "Queue a celery request for each page from chronicling america to warm up the cache."
 
     def handle(self, **options):
         for page in Page.objects.all():
-	    try:
+            try:
                 url = "https://chroniclingamerica.loc.gov%s" % page.url
-                _logger.info("queuing url %s"% url)
+                _logger.info("queuing url %s" % url)
                 tasks.load_page_in_cache.delay(url)
             except Exception, e:
                 _logger.exception(e)
```